src/components/scheduling/goals/SchedulingGoal.svelte

```diff
@@ -87,12 +126,34 @@
     });
   }
 
+  function duplicateGoalInvocation() {
+    dispatch('duplicateGoalInvocation', {
```

Comment:
the `dispatch` type declaration should be updated to include these new events

Reply:
Change has been made